src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java

```diff
@@ -302,7 +302,114 @@ public static boolean isOfType(int type, int... types) {
+    }
+
+    /**
+     * Determines if the token type belongs to one of the given type.
+     *
+     * @param type the Token Type to check
+     * @param first the acceptable token
+     *
+     * @return true if type equals to first
+     */
+    public static boolean isOfType(int type, int first) {
+        return type == first;
+    }
+
+    /**
+     * Determines if the token type belongs to the given type.
+     *
+     * @param ast the AST node to check
+     * @param first the acceptable token
+     *
+     * @return true if type matches to given type.
+     */
+    public static boolean isOfType(DetailAST ast, int first) {
+        return isOfType(ast.getType(), first);
+    }
+
+    /**
+     * Determines if the token type belongs to one of the given types.
+     *
+     * @param type the Token Type to check
+     * @param first the acceptable token
+     * @param second the acceptable token
+     *
+     * @return true if type equals to first or second
+     */
+    public static boolean isOfType(int type, int first, int second) {
+        return type == first || type == second;
+    }
+
+    /**
+     * Determines if the token type belongs to one of the given types.
+     *
+     * @param ast the AST node to check
+     * @param first the acceptable token
+     * @param second the acceptable token
+     * @return true if type matches one of the given types.
+     */
+    public static boolean isOfType(DetailAST ast, int first, int second) {
+        return isOfType(ast.getType(), first, second);
+    }
+
+    /**
+     * Determines if the token type belongs to one of the given types.
+     *
+     * @param type the Token Type to check
+     * @param first the acceptable token
+     * @param second the acceptable token
+     * @param third the acceptable token
+     *
+     * @return true if type equals to first or second or third
+     */
+    public static boolean isOfType(int type, int first, int second, int third) {
+        return type == first || type == second || type == third;
+    }
+
+    /**
+     * Determines if the token type belongs to one of the given types.
+     *
+     * @param ast the AST node to check
+     * @param first the acceptable token
+     * @param second the acceptable token
+     * @param third the acceptable token
+     * @return true if type matches one of the given types.
+     */
+    public static boolean isOfType(DetailAST ast, int first, int second, int third) {
+        return isOfType(ast.getType(), first, second, third);
+    }
+
+    /**
+     * Determines if the token type belongs to one of the given types.
+     *
+     * @param ast the AST node to check
+     * @param first the acceptable token
+     * @param second the acceptable token
+     * @param third the acceptable token
+     * @param fourth the acceptable token
+     * @return true if type matches one of the given types.
+     */
+    public static boolean isOfType(DetailAST ast, int first, int second,
```

Comment:
please group all `isOfType(DetailAST ast, .....` 
and then all `boolean isOfType(int type,....`

Reply:
Changes Done

Reply:
I am currently investigating what @baratali said here: https://github.com/checkstyle/checkstyle/pull/11543#issuecomment-1120026036 
Please give me some time to work on this issue.

Reply:
Report with latest commit: https://rahulkhinchi03.github.io/
(for isOfType() with first parameter)

Reply:
@romani  @baratali Updated Report with latest commit: https://rahulkhinchi03.github.io/
(for isOfType() with first parameter)